src/Compilers/VisualBasic/Test/Symbol/SymbolsTests/Metadata/PE/LoadingMethods.vb

```diff
@@ -925,5 +925,35 @@ BC30657: 'VT' has a return type that is not supported or parameter types that ar
                             ~~
 ]]>)
         End Sub
+
+        <Fact, WorkItem(217681, "https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_workitems?id=217681")>
+        Public Sub LoadingMethodWithPublicAndPrivateAccessibility()
+            Dim source =
+         <compilation>
+             <file>
+Class D
+   Shared Sub Main()
+      Dim test = new C()
+      test.M()
+      System.Console.WriteLine(test.F)
+
+      Dim test2 = new C.C2()
+      test2.M2()
+   End Sub
+End Class
+    </file>
+         </compilation>
+
+            Dim references = {MetadataReference.CreateFromImage(TestResources.SymbolsTests.Metadata.PublicAndPrivateFlags)}
+
+            Dim comp = CreateCompilationWithMscorlib(source, references:=references)
+            comp.VerifyDiagnostics(
```

Comment:
@jaredpar This is the preferred way to baseline diagnostics for VB (human readable with squiggle). See the previous unit-test, for example.